tests/rendering/worlds/.normals.wbproj

```diff
@@ -0,0 +1,9 @@
+Webots Project File version R2020b
```

Comment:
Maybe a gitignore in the '/rendering/worlds/' directory like for the other category (https://github.com/cyberbotics/webots/blob/590a26b182f46189d3ccc96e89fecdb872c43060/tests/protos/worlds/.gitignore) is cleaner?